src/workerd/util/sqlite.c++

```diff
@@ -4,7 +4,12 @@
 
 #include "sqlite.h"
 #include <kj/debug.h>
+
+#if _WIN32
+#else
```

Comment:
```suggestion
#if !_WIN32
```

Reply:
Also all the places where you've used `#ifndef`, can we use `#if !` instead? It's what KJ does. (The preprocessor treats undefined symbols as falsy.)